### DISCUSSION TITLE
How can I change the wallpaper?

### DISCUSSION THREAD

**Discussion by rrmt23** [enchant97/web-portal#91] [Q&A]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Is This A Security Issue?

- [X] This is not a security issue

### Describe The Problem

Hi

How can I change the wallpaper?
In the screenshot, I see that you have other wallpapers.

### Expected Behavior

Thank you!

### Environment

docker

**Comment by enchant97** [comment]
If you are referring to the light/dark theme. These can be changed via the sun and moon icon at the top right. This feature will require JavaScript.

Apart from that there is no other way of adding a custom background unless you use custom CSS, which is supported by uploading as an admin user.
